protos/ensembl/production/metadata/ensembl_metadata.proto

```diff
@@ -45,6 +45,9 @@ service EnsemblMetadata {
   // Get top level statistics
   rpc GetTopLevelStatistics(OrganismIDRequest) returns (TopLevelStatistics) {}
 
+  // Get top level statistics by UUID
+  rpc GetTopLevelStatisticsByUUID(GenomeUUIDRequest) returns (TopLevelStatistics) {}
```

Comment:
@kamaldodiya - it needs to return TopLevelStatisticsByUUID

Reply:
Good Spot @sanjay-boddu  Corrected now